qiskit/providers/ibmq/api_v2/ibmqversionfinder.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Client for accessing IBM Q's version finder."""
+
+from .session import RetrySession
+from .rest import Auth
+from .rest.version_finder import VersionFinder
+
+
+class IBMQVersionFinder:
+    """Client for finding the API version being used."""
+
+    def __init__(self, url, proxies=None):
+        """IBMQVersionFinder constructor.
+
+        Args:
+            url (str): URL for the service.
+            proxies (dict): proxies used in the connection.
+        """
+        self.url = url
+        self.client_auth = Auth(RetrySession(url, proxies=proxies))
```

Comment:
Can you remove this attribute, as mentioned in https://github.com/Qiskit/qiskit-ibmq-provider/pull/140#discussion_r291089189?